CustomLookupTest/test.cs

```diff
@@ -0,0 +1,286 @@
+using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+using Tests;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Linq.Expressions;
+using System.Reflection.Metadata;
+using System.Threading;
+using System.Timers;
+using System.Linq;
+using System.Runtime.ExceptionServices;
+/// <summary>
+/// This is my attempt at a unit test. I took information from VSE documentation and the team's current SkillsetsTest.cs
+/// </summary>
+namespace Test
+{
+
+    [TestClass]
+    public class CustomLookupTests
+    {
+        private static readonly HttpClient client = new HttpClient();
+        
+        [TestMethod]
+        public void CustomLookupTest1()
+        {
+            // tests against incorrect input (missing words)
+            string content = Constants.inputTest1;
+            HttpContent jsonContent = new StringContent(content, null, "application/json");
+            var response = client.PostAsync("http://localhost:7071/api/CustomEntitySearch", jsonContent).Result;
+            string responseString = response.Content.ReadAsStringAsync().Result;
+            Assert.AreEqual("Bad Request", response.ReasonPhrase, false);
+            Assert.AreEqual(Constants.checkTest1, responseString, false);
+        }
+
+        [TestMethod]
+        public void CustomLookupTest2()
+        {
+            // tests against incorrect input (missing text)
+            string content = Constants.inputTest2;
+            HttpContent jsonContent = new StringContent(content, null, "application/json");
+            var response = client.PostAsync("http://localhost:7071/api/CustomEntitySearch", jsonContent).Result;
+            string responseString = response.Content.ReadAsStringAsync().Result;
+            Assert.AreEqual("Bad Request", response.ReasonPhrase, false);
+            Assert.AreEqual(Constants.checkTest2, responseString, false);
+        }
+
+        [TestMethod]
+        public void CustomLookupTest3()
+        {
+            // tests against empty string text
+            string text = Constants.inputElement.Replace("#REPLACE ME#", @"""""");
+            string words = text.Replace("#INSERT WORDS#", Constants.inputTest3);
+            string content = Constants.inputCheckTest.Replace("#REPLACE ME#", words);
```

Comment:
>REPLACE ME [](start = 64, length = 10)

nitish: it might be more maintainable to refactor these into a helper methods

eg:

public static string GetContentPaylaod(string words)
{
    return $@"{{""values"":[{words}]}}"
}